infrastructure/src/main/java/org/corfudb/infrastructure/ServerContext.java

```diff
@@ -248,6 +249,7 @@ ChannelImplementation getChannelImplementation() {
      */
     public CorfuRuntimeParameters getDefaultRuntimeParameters() {
         return CorfuRuntime.CorfuRuntimeParameters.builder()
+                .priorityLevel(PriorityLevel.HIGH)
```

Comment:
Should this priority level be NORMAL by default?

Reply:
No, these Runtimes are constructed for the management servers and the orchestrator, since they do reconfiguration tasks they should be high priority.

Reply:
Maybe use another name other than "default" to avoid confusion?

Reply:
Done. Changed it to `getManagementRuntimeParameters`